distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -33,19 +38,15 @@ a {
 }
 
 .filters {
-  background-color: $green;
-  margin-right: $filters-margin-right;
-  padding-left: 40px;
-  padding-right: @padding-left;
-  min-width: $filters-width;
-
-  /* Make the filter sidebar touch the top of the page, but keep the text
-     aligned with the main content. */
-  margin-top: -($header-height)
-  padding-top: $header-height;
+  position: absolute;
+  width: $filters-width;
+  top: 10px;
+  left: 0;
+  padding: $header-height 20px 20px 0;
+  height: 100%;
```

Comment:
It looks like this, combined with the `border-box`, is giving the page a vertical scrollbar. I played around with getting rid of `border-box`. It should be possible to do that and keep the code elegant if we're clever.